etc/config.xml

```diff
@@ -59,6 +59,8 @@
                 <can_use_internal>1</can_use_internal>
                 <can_use_checkout>1</can_use_checkout>
                 <is_gateway>1</is_gateway>
+                <privateInfoKeys></privateInfoKeys>
+                <paymentInfoKeys>buckaroo_original_transaction_key</paymentInfoKeys>
```

Comment:
what's the use of these 2 tags?

Reply:
I added a comment on the ticket: https://buckaroonl.atlassian.net/browse/BP-2945?focusedCommentId=101658

"Steps to add another field from payment additional information to Order View from admin with the new method: 

Add in paymentInfoKeys the key from additional information that you want to be included. Pay attention if you added also in privateInfoKeys it will not appear."